enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/components/Title.tsx

```diff
@@ -0,0 +1,12 @@
+import { QuestionTitle } from "embedding-sdk/components/private/QuestionTitle";
+import { useInteractiveQuestionContext } from "embedding-sdk/components/public/InteractiveQuestion/context";
+
+export const Title = () => {
+  const { customTitle, question, withTitle } = useInteractiveQuestionContext();
+
+  return (
+    question &&
+    withTitle &&
+    (customTitle || <QuestionTitle question={question} />)
+  );
```

Comment:
You don't need a question if you're using `customTitle`
```suggestion
  return (
    withTitle &&
    (customTitle || (question && <QuestionTitle question={question} />))
  );
```